tests/asconfig/entry-points/assembly/globals.ts

```diff
@@ -0,0 +1,3 @@
+@global
+  const answerToLife = 42;
```

Comment:
```suggestion
@global
const answerToLife = 42;
```

Reply:
This causes the linter to fail, despite looking wrong.

Reply:
Oh, I believe I fixed that in other places by doing

```ts
// @ts-ignore: decorator
@global const answerToLife = 42;
```